{{cookiecutter.module_name|slugify}}/.gitignore

```diff
@@ -1,5 +1,5 @@
-inmanta_plugins/**/__pycache__
-tests/**/__pycache__
+plugins/__pycache__
+tests/__pycache__
```

Comment:
Why is this change required?

Reply:
This looks like a mistake in merging v2 into this branch.